src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -0,0 +1,2122 @@
+            }
+        }
+
+        /// <summary>
+        /// Gets the microphone volume during recording.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <pre>
+        /// The Service State must be Recording.
+        /// </pre>
+        public static float RecordingVolume
+        {
+            get
+            {
+                float volume;
+
+                ErrorCode error = VcMgrGetRecordingVolume(out volume);
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "RecordingVolume Failed with error " + error);
+                    return 0;
+                }
+
+                return volume;
+            }
+        }
+
+        /// <summary>
+        /// Select valid result from all results.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <param name="list">The valid result list</param>
+        /// <exception cref="InvalidOperationException">
+        /// This can occur due to the following reasons:
+        /// 1) This exception can be due to an invalid state.
+        /// 2) This exception can be due to operation failed.
+        /// 3) This exception can be due to out of memory.
+        /// </exception>
+        public static void SetSelectedResults(VoiceCommandList list)
+        {
+            ErrorCode error = VcMgrSetSelectedResults(list._handle);
+            if (error != ErrorCode.None)
+            {
+                Log.Error(LogTag, "SetSelectedResults Failed with error " + error);
+                throw ExceptionFactory.CreateException(error);
+            }
+        }
+
+        /// <summary>
+        /// Called when client gets the all recognition results from vc-daemon.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <pre>
+        /// The State should be Initialized
+        /// </pre>
+        public static event EventHandler<SetAllResultEventArgs> SetAllResult
```

Comment:
Why `Set`? I can't understand
it was called when recognition data was delivered from vc-daemon.
So, it more proper `ResultUpdated` or `Recognized`

Reply:
all result is the result of matching the command.
This result is sent from VC service.

Simple example
1. A music app, B music app exists.
and A music app can be (Stop, Play command), B music app can be (Stop, Next Command).
2. Say 'Stop music'.
3. VC manager can get two matching results in SetAllResult callback.
4. VC manager app can choose one of the two results. and VC manager app can even cancel all the results.

Next.
1. Say ''Next music'.
2. VC manager can get one matching results.
3. VC manager app can select or cancel matched results.

Reply:
This is the event to delivery all matched Commands.
How about rename to 
```suggestion
        public static event EventHandler<SetAllResultEventArgs> RecognizedCommandsDelivered
```

Reply:
And, If VoiceControlManager has responsibility to select a command,
It better to use delegate instead of event

``` c#
 delegate VoiceCommand RecognizedCommandsSelection(IEnumerable<VoiceCommand> commands);
....
  public void SetRecognizedCommandsSelectionDelegate(RecognizedCommandsSelection delegate);
```

Reply:
Fixed.